unity-renderer/Assets/DCLPlugins/RealmsPlugin/RealmBlockerModifier.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using Decentraland.Bff;
+
+namespace DCLPlugins.RealmPlugin
+{
+    public class RealmBlockerModifier : IRealmModifier
```

Comment:
What do you think about adding a short XML summary. Just to main responsibility of the class?

Reply:
Done!